answer/answer.json

```diff
@@ -19,7 +19,10 @@
 			"description": "In case of negative answer, error generated by the client query",
 			"$ref": "negativeAnswer/error.json"
 		},
-		"id": { "type": "integer" }
+		"id": { 
+			"type": "integer",
+			"$comment": "The id match the request id. If there was an error in detecting the id, it must be Null"
+		      }
```

Comment:
The idea proposed in #91 is very good! It will probably simplify some steps in error handling and doesn't require a pre-agreed dummy value (e.g. `-1` for missing id). The only problem with what you wrote, it that if someone tries to use the JsonSchema, a NULL value won't be accepted. I would suggest modiying your Json-Schema to look like this:

```json
"id": {
    "oneOf": [
        { "type" : "integer" },
        { "type": null  }
    ],
    "$comment": "... // your comment"
}
```
See this link for more information on [Json-Schema null](https://json-schema.org/understanding-json-schema/reference/null.html)

Reply:
Well spotted. I completely missed this 😅 Thanks @RaulinN